lib/archethic.ex

```diff
@@ -137,6 +137,35 @@ defmodule Archethic do
     TransactionChain.resolve_last_address(address)
   end
 
+  @doc """
+  Retrieve the last transaction address for a chain locally
+  It queries the the netowork for genesis address
+  """
+  @spec get_last_transaction_address_locally(address :: binary()) ::
+          {:ok, binary()} | {:error, :not_found} | {:error, :network_issue}
+  def get_last_transaction_address_locally(address) when is_binary(address) do
```

Comment:
This function may be in TransactionChain module, same for `get_genesis_transaction_address`, `fetch_transaction_chain_locally` and  `available_loccaly?` as we request an information of a transaction chain